wiki/en/Privacy-Statement.md

```diff
@@ -7,32 +7,38 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
+## Definition of Terms
+
+- "**Server**" The Jamulus server software, as opposed to the host machine/OS it is running on.
```

Comment:
I'd add "in general" unless you've been _incredibly_ careful :).

Reply:
Where would "in general" go here?

Reply:
Maybe add a note after the definition saying "There may be some places in the documentation where 'server' has been used to refer to the system hosting the Jamulus Server, rather than the Server itself.  Please report these and we'll tidy them up!" :)

Reply:
I'm going to do a pass at the end for that. We've been using capitalised words to denote "specific" meanings like "Server" to denote the software, and lowercase to denote generic terms like the system it runs on. Same for Directory, for example.